docs/source/install/deb.rst

```diff
@@ -25,8 +32,11 @@ Setup repositories
 
   .. code-block:: bash
 
+    # Pick OS version from: trusty, jessie, wheezy
+    export DISTRO=trusty
     wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
-    echo "deb https://dl.bintray.com/stackstorm/`lsb_release -cs`_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
```

Comment:
I still say we hardcode it.  That way if we don't generate a repo for a code name it will fall back to trusty.